src/main/kotlin/sh/hsp/techtree/graphviz/DSLConverter.kt

```diff
@@ -0,0 +1,26 @@
+package sh.hsp.techtree.graphviz
+
+import sh.hsp.techtree.TreeModel
+import sh.hsp.techtree.TreeNode
+import java.util.stream.Collectors
+
+class DSLConverter {
+    fun convert(model: TreeModel): String {
+         val nodesAsDSL = model.nodes.stream()
+            .map { registerNode(it) }
+            .collect(Collectors.joining(" "))
+
+        return "digraph { $nodesAsDSL }"
```

Comment:
https://kotlinlang.org/docs/java-to-kotlin-idioms-strings.html#use-multiline-strings
Możesz tu enter dodać, by były ładne paddingi

Reply:
Możesz też użyć $nodesAsDSL.padStart(4), by dodać wcięcia